CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Using methods only available for Query-based Realms now throw a better error message if called on the wrong Realm type.
```

Comment:
I don't think our documentation talks about "Realm type". Maybe it can be confused with "Realm property type"? Is "Realm file type" or "Realm instance type" a better term?

Reply:
Realm file type sounds fine with me